### PR TITLE
Add a guide of how to use git

### PR DESCRIPTION
**Pull request by EmmaNguyen**
Hi Giang,

I would like you to review my outline of how to use git hub. 

Please let me know if you see any missing here.

Thanks

### CODE REVIEW

File: guides/how_to_use_git.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+### Overview
+
+In this tutorial, you will learn step-by-step of how to use git as follow:
+
+- How to create an account for git
```
Comment (giang-nghg) [inline guides/how_to_use_git.md]: It should be "for github", as git doesn't need an account, only name & email to put as author on commits

File: guides/how_to_use_git.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- How to create a repo on git
```
Comment (giang-nghg) [inline guides/how_to_use_git.md]: Similarly, I think you mean "on github"? Maybe we should present how to create & manage a repo locally first, then introduce github once we move to sharing repo, collaborating, etc.?

**Review by giang-nghg — CHANGES_REQUESTED** [review]
We should not let readers confuse github to be a part of git

File: guides/how_to_use_git.md (hunk 1 above)
Comment (EmmaNguyen) [reply guides/how_to_use_git.md]: Agree.

File: guides/how_to_use_git.md (hunk 2 above)
Comment (EmmaNguyen) [reply guides/how_to_use_git.md]: Solved.

**Comment by EmmaNguyen** [comment]
- Change from git to github

**Pull request merged**